### PR TITLE
AC Automated Test - LRAC-8321 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**
rebase for bchan
https://github.com/brianchandotcom/liferay-portal/pull/105836

### CODE REVIEW

**Comment by alexcandidos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path [hunk 1]
```diff
@@ -55,7 +55,7 @@
 	<td></td>
 </tr>
 <tr>
-	<td>AVAILABLE_NO_PROPERTIES_FOUND</td>
+	<td>NO_PROPERTIES_FOUND</td>
 	<td>//span[contains(@class,'title') and text()='No Properties Found']</td>
 	<td></td>
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path:60]: hi @rafaellajordao , could you please move this path to `ACSettings.path` ?

Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path]: fixed @alexcandidos  !! sorry about this!!

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
ci:forward

**Comment by alexcandidos** [comment]
ci:stop

**Comment by alexcandidos** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, could you try close this PR then resend a new one? It probably be stuck by some reason.

**Comment by alexcandidos** [comment]
yeah @Tim-Cao , makes sense! 
Just resent via https://github.com/liferay-qa/liferay-portal/pull/780. Thanks!

**Pull request closed without merging**